File: Patterns/python/pattern7.py
# n = 4

#    1
#   23
#  456
# 78910

n = int(input())
row = 1
val = 1
while row <= n :
    space = 1
    while space <= n - row:
        print(" ", end="")
        space += 1
    
    col = 1
    while col <= row:
        print(val, end="")
        col += 1
        val += 1
    row += 1
    print("")

File: Patterns/go/pattern7.go
// n = 4

// ****
// ****
// ****
// ****

package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)
	
	row := 1
	for row <= n {
		col := 1
		for col <= n {
			fmt.Printf("*")
			col++
		}
		row++
		fmt.Printf("\n")
	}
}